packages/next/build/webpack/plugins/nextjs-require-cache-hot-reloader.ts

```diff
@@ -47,8 +46,6 @@ const PLUGIN_NAME = 'NextJsRequireCacheHotReloader'
 export class NextJsRequireCacheHotReloader implements WebpackPluginInstance {
   prevAssets: any = null
   hasServerComponents: boolean
-  previousOutputPathsWebpack5: Set<string> = new Set()
```

Comment:
Had a look at these and it turns out it only adds / removes and never reads the set so it was not in use. Same for the line below.